app/src/lib/open-shell.ts

```diff
@@ -15,14 +15,20 @@ export function openShell(fullPath: string, shell?: string) {
     return spawn('START', [shell || 'cmd'], { shell: true, cwd: fullPath })
   }
 
+  if (__LINUX__) {
+    const commandArgs = ['--working-directory', fullPath]
+    return spawn('gnome-terminal', commandArgs, { shell: true })
```

Comment:
Yes, probably. Run traditionally under GNOME it should actually look at a particular GSetting instead of assuming `gnome-terminal`. Run inside a Flatpak sandbox, it should be using a "portal" to request the system to open a terminal. Those are the only two ways I'm familiar with but I'm sure KDE, AppImage and other environments / sandboxes have different ways as well.